### PR TITLE
Prep for v1.1.5 release

### LINKED ISSUE #141: Can not install schema-enforcer in environments which require a version of jsonschema < 4.6
### Environment
* Python version:   3.7.10
* schema-enforcer version:   1.1.4

### Expected Behavior
I expected to be able to install schema-enforcer in an environment where the jsonschema library could be anything > 3.2 and <5.

### Observed Behavior
When the version of jsonschema installed is < 4.6, the `format-nongpl` extra can't be found.

This looks to be because `format_nongpl` was changed to `format-nongpl` in jsonschema 4.6.

 https://github.com/python-jsonschema/jsonschema/issues/962

### PR DESCRIPTION
**Pull request by PhillSimonds**
Fixes #141

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.1.5 - TBD
```
Comment (chadell) [inline CHANGELOG.md:3]: date?

Comment (PhillSimonds) [reply CHANGELOG.md:3]: Hopefully `2022-07-25` -- was planning to add that into develop directly when we cut a release.

**Comment by glennmatthews** [comment]
Is this the right fix? Do we have any environments that need `jsonschema>=4.6` at this point in time, and so would be incompatible here?

**Comment by PhillSimonds** [comment]
It is the right fix. We have environments

> Is this the right fix? Do we have any environments that need `jsonschema>=4.6` at this point in time, and so would be incompatible here?

I believe it is the right fix -- v1.1.4 added support for jsonschema versions in the 4.x train. It was removed fom pypy and github because of this issue. We are planning to release this version (v1.1.5) which will allow for `jsonschema<4.6`, then another one at the same time (v1.2.0) which will allow for `jsonschema>=4.6`.

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**